tests/cargo-rm.rs

```diff
@@ -19,6 +19,19 @@ fn remove_existing_dependency() {
     assert_eq!(toml["features"]["std"].as_array().unwrap().len(), 2);
 }
 
+#[test]
+fn remove_existing_dependency_does_not_create_empty_tables() {
+    let (_tmpdir, manifest) = clone_out_test("tests/fixtures/rm/Cargo.toml.no_features.sample");
+
+    let toml = get_toml(&manifest);
+    assert!(toml["features"].is_none());
+    assert!(toml["build-dependencies"].is_none());
+    execute_command(&["rm", "docopt"], &manifest);
```

Comment:
I think we should test removing an optional dependency which triggers feature removal, no?
```suggestion
    execute_command(&["rm", "clippy"], &manifest);
```